Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -301,8 +302,9 @@
     <string name="risk_details_headline_period_logged">"Period Logged"</string>
     <!-- XHED: risk details - infection period logged headling, below behaviors -->
     <string name="risk_details_subtitle_period_logged">"This period is included in the calculation."</string>
+    <!-- XTXT: Tracing explanation dialog message -->
+    <string name="tracing_explanation_dialog_message">"Die Berechnung des Infektionsrisikos kann nur für die Zeiträume erfolgen, an denen die Risiko-Ermittlung aktiv war. Die Risiko-Ermittlung sollte daher dauerhaft aktiv sein. Für Ihre Risiko-Ermittlung wird der Zeitraum der letzten %d Tage betrachtet.\nÄltere Tage werden automatisch gelöscht, da sie aus Sicht des Infektionsschutzes nicht mehr relevant sind.\nWeitere Informationen finden Sie in den FAQ."</string>
```

Comment:
Why german? Shouldn't this stay empty for translation?

Reply:
I've created a new string, and it needs to be present in the default strings file as well. The actual text will get overwritten by the translation anyway.